### PR TITLE
Added view-colour indicator to cells within view-hierarchy table view controller.

### PR DESCRIPTION
**Pull request by terry-au**
This PR adds an indicator for the background colour of a view in the view hierarchy table.

I've attached a screenshot of the functionality below:
Views with background colour indicators..

This has been a pretty useful feature to me, as it allows quickly locating desired views, based on colour.

### CODE REVIEW

File: Classes/ViewHierarchy/FLEXHierarchyTableViewController.m [hunk 1]
```diff
@@ -180,6 +181,20 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
     
+    // Use a pattern-based colour to simplify application of the checker pattern.
+    static UIColor *checkerPatternColour = nil;
+    static dispatch_once_t once;
+    dispatch_once(&once, ^{
+        checkerPatternColour = [UIColor colorWithPatternImage:[FLEXResources checkerPattern]];
+    });
+    
+    UIColor *viewColour = view.backgroundColor;
+    if (!viewColour || [viewColour isEqual:[UIColor clearColor]]){
```
Comment (NSExceptional) [inline Classes/ViewHierarchy/FLEXHierarchyTableViewController.m]: Code style: missing space between `) {`
```suggestion
    if (!viewColour || [viewColour isEqual:[UIColor clearColor]]) {
```

File: Classes/ViewHierarchy/FLEXHierarchyTableViewController.m [hunk 2]
```diff
@@ -180,6 +181,20 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
+        cell.viewBackgroundColorView.backgroundColor = checkerPatternColour;
+    }else{
```
Comment (NSExceptional) [inline Classes/ViewHierarchy/FLEXHierarchyTableViewController.m]: Code style: missing spaces surrounding `else`
```suggestion
    } else {
```

**Comment by NSExceptional** [comment]
Nice! A few things though:

Could you fix it so that the text of the cells truncates when it gets to the bubble?

It's not apparent to me why your changes here are necessary, along with the changes in `setSelected:`. What happens if you don't do this?

```objc
- (void)setHighlighted:(BOOL)highlighted animated:(BOOL)animated
{
    UIColor *originalColour = self.viewBackgroundColorView.backgroundColor;
    [super setHighlighted:highlighted animated:animated];
    ...
    self.viewBackgroundColorView.backgroundColor = originalColour;
}
```

**Comment by terry-au** [comment]
I’ll add the changes you requested. The setHighlighted and setSelected changes prevent the colour within the indicator from disappearing when the cell is selected or highlighted. It’s kind of useful when the view hierarchy table highlights the selected view, for confirmation of the selected view’s colour.

**Comment by terry-au** [comment]
Updated to fix the overlapping label issue.

**Comment by NSExceptional** [comment]
Looks good! One last thing, could you squash these all into just one or two commits? The maintainers like to keep the commit history relatively tidy (small commits like `Update blah` are undesirable)

Then we're good to merge :)

**Comment by terry-au** [comment]
Commits have been merged into one single commit, as requested. Should be clear to merge.

**Pull request merged**